src/main/java/org/apache/mesos/offer/MesosResourcePool.java

```diff
@@ -219,7 +251,8 @@ private boolean sufficientValue(Value desired, Value available) {
         return mesRsrcs;
     }
 
-    private Map<String, MesosResource> getReservedPool(Offer offer) {
+    private static Map<String, MesosResource> getReservedPool(
+            Collection<MesosResource> mesosResources) {
```

Comment:
Since this method is private, it'll only ever be able to operate on `this.mesosResources` anyway. Is the parameter here for testability? If so, great, otherwise, it could be dropped.